src/Common/Helpers/BrokeredMessageExtensions.cs

```diff
@@ -28,14 +28,19 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
+using System.ServiceModel.Channels;
+using System.Xml;
 
 #endregion
 
 namespace ServiceBusExplorer.Helpers
 {
     public static class BrokeredMessageExtensions
     {
+        public const string UnableToReadMessageBody = "Unable to read the message body.";
+
         private static readonly PropertyInfo BodyStreamPropertyInfo;
+        private const int MaxBufferSize = 262144; // 256 KB
```

Comment:
```suggestion
        const int MaxBufferSize = 1024 * 256; // 256 KB
```

Alternatively, could be done in the following way:

const int MaxBufferSizeInKB = 1024 * 256;

Reply:
Happy to change the calc - kept it as private as was in `LogBrokeredMessageInspector` and not used elsewhere so not sure about increasing the scoped